mbedtls/src/bignum/mod.rs

```diff
@@ -407,6 +408,21 @@ impl Mpi {
 
         Ok(r)
     }
+
+    /// The Miller-Rabin primality test. Returns `MpiNotAcceptable` is self is
+    /// definitely not prime. If no error is returned, self is probably prime. See
+    /// mbedtls_mpi_is_prime.
+    pub fn is_prime<F: Random>(&self, rng: &mut F) -> Result<()> {
+        unsafe {
+            mpi_is_prime(
```

Comment:
The `mbedtls_mpi_is_prime` function is deprecated in favor of `mbedtls_mpi_is_prime_ext` that requires specifying the number of rounds of the Miller-Rabin test. I'm wondering if we should provide the same functionality.

Reply:
I missed that, thanks for pointing this out. Yes, definitely.